src/core/core.js

```diff
@@ -121,6 +121,9 @@ export default class Core {
      * @type {ComponentManager}
      */
     this._componentManager = config.componentManager;
+
+    /** @type {{ [agent: string]: string, ANSWERS_CORE?: never, ANSWERS_SEARCH_UI_SDK?: never }} */
```

Comment:
I see that in answers-core, CustomClientSDK have the `agent` field as `[agent: string]: string | undefined`, can it be undefined?

Reply:
yeah, I'll update after Rose gets back with the updated config name